build.gradle

```diff
@@ -247,6 +247,14 @@ configure(opentelemetryProjects) {
         links 'https://docs.oracle.com/javase/8/docs/api/'
     }
 
+    if (JavaVersion.current().isJava8Compatible()) {
+        allprojects {
+            tasks.withType(Javadoc) {
```

Comment:
Instead of tasks.withType(Javadoc), why not add the addBooleanOption line to the existing `javadoc.options` block above?

Reply:
👍 you were right.